src/measure_theory/bochner_integration.lean

```diff
@@ -1146,6 +1146,69 @@ begin
     exact norm_integral_le_lintegral_norm _ }
 end
 
+/-- Lebesgue dominated convergence theorem for filters with a countable basis -/
+lemma tendsto_integral_filter_of_dominated_convergence {ι} {l : filter ι}
+  {F : ι → α → β} {f : α → β} (bound : α → ℝ)
+  (hl_cb : l.has_countable_basis)
+  (F_measurable : ∀ n, measurable (F n))
+  (f_measurable : measurable f)
+  (h_bound : ∀ n, ∀ₘ a, ∥F n a∥ ≤ bound a)
+  (bound_integrable : integrable bound)
+  (h_lim : ∀ₘ a, tendsto (λ n, F n a) l (𝓝 (f a))) :
+  tendsto (λn, ∫ a, F n a) l (𝓝 $ (∫ a, f a)) :=
+begin
+  rw filter.tendsto_iff_seq_tendsto,
+  { intros x hxl,
+    refine tendsto_integral_of_dominated_convergence _ _ _ _ _ _,
+    { exact bound },
+    { intro, apply F_measurable },
+    { assumption },
+    { assumption },
+    { intro, apply h_bound },
+    { filter_upwards [h_lim],
+      simp only [mem_set_of_eq],
+      assume a h_lim,
+      apply @tendsto.comp _ _ _ x (λ n, F n a);
+      assumption } },
+  assumption
+end
+
+/-- Lebesgue dominated convergence theorem for filters with a countable basis -/
```

Comment:
Please explain the difference between these two lemmas in their docstrings.

Reply:
BTW, can you deduce the previous version from this one using `univ_mem_sets' F_measurable` and `univ_mem_sets' h_bound`?

Reply:
That's true. Maybe I'll just remove the previous version since it is not really doing much.